### PR TITLE
coprocessor/xeval: add function signatures mapping

### PR DESCRIPTION
**Pull request by hhkbp2**
Hi,

This PR adds the mapping fro function signatures to implementation function for evaluator.

PTAL @XuHuaiyu @AndreMouche @lishihai9017 @andelf

### CODE REVIEW

File: src/coprocessor/xeval/evaluator.rs [hunk 1]
```diff
@@ -521,6 +521,11 @@ impl Evaluator {
     fn eval_scalar_function(&mut self, ctx: &EvalContext, expr: &Expr) -> Result<Datum> {
         match expr.get_sig() {
             ScalarFuncSig::AbsInt => self.abs_int(ctx, expr),
+            ScalarFuncSig::AbsReal => self.abs_real(ctx, expr),
+            ScalarFuncSig::CeilInt => self.ceil_int(ctx, expr),
```
Comment (andelf) [inline src/coprocessor/xeval/evaluator.rs]: Ceiling & floor.

Comment (AndreMouche) [inline src/coprocessor/xeval/evaluator.rs]: In mysql `CEIL()` is a synonym for `CEILING()`(https://dev.mysql.com/doc/refman/5.7/en/mathematical-functions.html#function_ceil) , I think is ok to use `ceil` here. @andelf

**Comment by andelf** [comment]
LGTM

File: src/coprocessor/xeval/evaluator.rs (hunk 1 above)
Comment (hhkbp2) [inline src/coprocessor/xeval/evaluator.rs]: @XuHuaiyu Is it `Ceil` ok? better than `Ceiling`?

Comment (XuHuaiyu) [inline src/coprocessor/xeval/evaluator.rs]: In TiDB, we support both ceil and ceiling, 
they totally the same behavior. @hhkbp2

**Pull request merged**